src/hidimstat/distilled_conditional_randomization_test.py

```diff
@@ -683,27 +612,25 @@ def d0crt(
     method="predict",
     estimated_coef=None,
     sigma_X=None,
-    params_lasso_screening={
-        "alpha": None,
-        "n_alphas": 10,
-        "alphas": None,
-        "alpha_max_fraction": 0.5,
-        "cv": 5,
-        "tol": 1e-6,
-        "max_iter": 1000,
-        "fit_intercept": False,
-        "selection": "cyclic",
-    },
-    params_lasso_distillation_x=None,
+    lasso_screening=LassoCV(
+        n_alphas=10,
+        cv=5,
+        tol=1e-6,
+        fit_intercept=False,
+    ),
+    model_distillation_x=LassoCV(
+        n_jobs=1,
+        n_alphas=10,
+    ),
     refit=False,
-    screening=True,
-    screening_threshold=1e-1,
+    screening_threshold=10,
     centered=True,
     n_jobs=1,
     joblib_verbose=0,
     fit_y=False,
-    random_state=2022,
     scaled_statistics=False,
+    random_state=2022,
```

Comment:
We should not set random_state inside the function.

Reply:
I set the default to None